### PR TITLE
Fix for google.*

### PR DESCRIPTION
**Pull request by Rakleed**
Examples: https://www.google.com, https://www.google.com/search?q=dark+reader

### CODE REVIEW

**Comment by Rakleed** [comment]
And why in the address of the site at the beginning of the site is www, if without this everything works correctly?

**Comment by Gusted** [comment]
And why changing it to a changing CSS selector? This will become a cat and mouse game with Google to stay up to date with this selector.

**Comment by Gusted** [comment]
Poof it's already changed again. That's why we use a CSS3 selector.

**Comment by Rakleed** [comment]
The rule `a[href*="about/products"]` inverts the icon on the site with the search result, although it does not need to be done there (for example, https://www.google.com/search?q=dark+reader).

**Comment by Gusted** [comment]
I see... That's wow Didn't even noticed.

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -5198,7 +5198,7 @@ INVERT
 .gb_0b
 #dictionary-modules img[src*="png"]
 a.gb_b > div
-a[href*="about/products"]
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:5201]: `a[href*="about/products"][title]` The google main page has a title the search one has aria-label. We should preserve this selector as going down to the normal class selector will be litterly cat and mouse.

**Comment by Rakleed** [comment]
> I see... That's wow Didn't even noticed.

It's sarcasm? After all, this is incorrect inversion.

> 
> Poof it's already changed again. That's why we use a CSS3 selector.

I still have the same name.
image

**Comment by Gusted** [comment]
Google and dynamic classes :)

No it wasn't sarcasm I never knew that the 9 dots on search one was already bright enough.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Rakleed) [reply src/config/dynamic-theme-fixes.config:5201]: Previously done by name (for example, `a[title="Google apps"]`, but in different language versions of the search this name will be in a different language.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:5201]: That's why we don't add the ="" But simple the `[title]` That way it just check if it exist.

**Comment by Rakleed** [comment]
@Gusted And what's about?
> And why in the address of the site at the beginning of the site is www, if without this everything works correctly?

**Comment by Gusted** [comment]
Sub Domains :)
docs.google.com
drive.google.com
etc etc etc

**Pull request merged**